src/unix/newlib/mod.rs

```diff
@@ -0,0 +1,566 @@
+use dox::mem;
```

Comment:
Right, I forgot to remove that earlier. I'll do that when I push another fix that I need to make.